AirCasting/Utils/SystemSettings.swift

```diff
@@ -0,0 +1,23 @@
+// Created by Lunar on 15/07/2021.
+//
+
+import Foundation
+import SwiftUI
+
+func goToLocationAuthSettings() {
```

Comment:
let's not use global functions without a *really* good reason. Please create some protocol+class pair for it and inject using environment objects where needed